src/javascript/app/pages/user/accounts.js

```diff
@@ -70,13 +70,7 @@ const Accounts = (() => {
     const populateExistingAccounts = () => {
         const all_login_ids = Client.getAllLoginids();
         all_login_ids
-            .filter(loginid => !Client.get('is_disabled', loginid) && !Client.get('excluded_until', loginid))
-            .sort((a, b) => a > b)
-            .forEach((loginid) => {
-                appendExistingAccounts(loginid);
-            });
-        all_login_ids
-            .filter(loginid => Client.get('is_disabled', loginid) || Client.get('excluded_until', loginid))
```

Comment:
we don't want to show a list of disabled and excluded accounts in account page anymore?

Reply:
@negarn I see. I couldn't figure out what this was for. I will add these checks back.

Reply:
we can add a comment explaining what they are for :)